class.jetpack-gutenberg.php

```diff
@@ -326,6 +326,20 @@ public static function get_jetpack_gutenberg_extensions_whitelist() {
 		return $preset_extensions;
 	}
 
+	/**
+	 * Returns a diff from a combined list of whitelisted extensions and extensions determined to be excluded
+	 *
+	 * @return array A list of blocks: eg [ 'publicize', 'markdown' ]
+	 */
+	public static function get_available_extensions() {
+		$exclusions             = get_option( 'jetpack_extensions_exclusions', array() );
```

Comment:
Since we're adding this option, will you add it to:

- The options whitelist in the `Jetpack_Sync_Defaults` class so that we can sync it and provide some support tools as necessary
- In the `Jetpack_Options::get_all_wp_options()` array so that we can clear the option out with our reset tools

Reply:
Thanks! updated and added to both.